src/components/MonitorSettingDialog.vue

```diff
@@ -77,7 +80,7 @@ export default {
                 name: monitor.element.name,
                 monitor_index: monitor.index,
                 group_index: group.index,
-                isClickAble: this.showLink(monitor),
+                isClickable: monitor.element.sendUrl,
```

Comment:
could you explain why removing `this.showLink` here (and below) is okay?
This seems somewhat sus.

Reply:
@CommanderStorm 
Thank you for the review!

* The following part will always be `false` because it's not called with `ignoreSendUrl = true`. (The same method exists in `PublicGroupList`. The method was probably just copied from there.)
https://github.com/louislam/uptime-kuma/blob/46d8744fa40cf6a5773578252af655058437b1e6/src/components/MonitorSettingDialog.vue#L108-L110

* The `monitor.element.url` is causing the issue reported. Therefore, I thought it would be sufficient to keep only `monitor.element.sendUrl`. The `url` only exists as an attribute of the `element` when `sendUrl` is saved with a value of 1 (true) to the database.
https://github.com/louislam/uptime-kuma/blob/46d8744fa40cf6a5773578252af655058437b1e6/src/components/MonitorSettingDialog.vue#L111
This results in the issue described in the report.
  1. If you open the dialog with `sendUrl` saved as 0 (false) and toggle `sendUrl` by using the checkbox (see [`toggleLink()`](https://github.com/louislam/uptime-kuma/blob/46d8744fa40cf6a5773578252af655058437b1e6/src/components/MonitorSettingDialog.vue#L92-L94)),
  2. since it has not been saved to the database yet,
  3. reopening the dialog will result `showLink()` to be `false` because the `element` has no `url` attribute.

(I’m not a native English speaker, so my explanation might be unclear. It would be helpful if you could check the operation on your end.)

Reply:
After I investigated your another comment, I thought that addressing it by removing the following might be a good approach. What do you think?
https://github.com/louislam/uptime-kuma/blob/46d8744fa40cf6a5773578252af655058437b1e6/server/model/monitor.js#L55-L57

I’ve created another branch in my repository. I plan to do it like this.
https://github.com/R-HNF/uptime-kuma/commit/ae5d0b2c7be6b37deb0c0ff94ccc54635ad847fe

Reply:
Looks much more sane 👍🏻
Lets also remove the unused if-statement surrounding `ignoreSendUrl` though.

Reply:
@CommanderStorm
I addressed it in these commits:
* d3c3808daec62c15aa39d8a943d47c38e8d9b9f5
* 061b549d16a02ff7213f482c2a4366f9776e0c2a

Could you re-review this PR? Thank you!